types/ramda/test/toPairsIn-tests.ts

```diff
@@ -1,13 +1,11 @@
 import * as R from 'ramda';
 
 class F {
-  [k: string]: string;
-  x = 'X';
-  y = 'Y';
+  x: 'X';
+  y: 'Y';
 }
 
 () => {
   const f = new F();
-  const a1 = R.toPairsIn(f); // => [['x','X'], ['y','Y']]
```

Comment:
Why is this test removed? Shouldn't it still work?

Reply:
Good catch !
I was unnecessarily introducing a breaking change.
I restored the old tests and fixed the issue. I did the same thing with `toPairs()`, there was the same problem.